SetupContextMenu.ps1

```diff
@@ -7,106 +7,115 @@ Param(
 # Global definitions
 $config = "$env:LocalAppData\Packages\Microsoft.WindowsTerminal_8wekyb3d8bbwe\LocalState\profiles.json"
 $resourcePath = "$env:LOCALAPPDATA\WindowsTerminalContextIcons\"
+$resourcePathReg = "%LOCALAPPDATA%\WindowsTerminalContextIcons\"
```

Comment:
`%Varible name%` is the BAT style environment variable.
This line is the same as $resourcePath.

Ref: [About Environment Variables](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7)

Reply:
Slightly different. The first expands the path to its full path. The second retains the variable name in the registry. I don't know why, but this allows the icons to load for me (without needing to log out and back in at least)

Reply:
You mean that putting the raw `%LOCALAPPDATA%\WindowsTerminalContextIcons\` into registry?

Reply:
Yeah (plus the icon name)

Reply:
Could you check if the expand one really no work?
Windows may cache the context menu icons.
So you have to re-login or reboot to deal with it.